File: Widgets/1-Toplevels/GtkWindow/gtkwindow.py
#!/usr/bin/python
# python gtkwindow.py  

import gi
gi.require_version('Gtk', '3.0')
from gi.repository import Gtk

window = Gtk.Window()

title = "Hello Gtk!"
window.set_title (title)
window.set_default_size (800, 400)
width, height = window.get_default_size ()
print ("Width: %d, Height: %d"% (width, height))

window.set_position (Gtk.WindowPosition.MOUSE)
width, height = window.get_position ()
print ("PosX: %d, PosY: %d"% (width, height))

window.connect("destroy", Gtk.main_quit)
window.show_all()

Gtk.main()

File: Widgets/1-Toplevels/GtkWindow/gtkwindow.js
#!/usr/bin/gjs
// gjs gtkwindow.js

imports.gi.versions.Gtk = '3.0';
const {GObject, Gtk} = imports.gi;

Gtk.init(null);

let window = new Gtk.Window();
let title = "Hello Gtk!";
window.set_title (title);

window.set_default_size (800, 400);
let [width, height] = window.get_default_size ();
print (`Width: ${width}, Height: ${height}`);

window.set_position (Gtk.WindowPosition.MOUSE);
[width, height] = window.get_position ();
print (`PosX: ${width}, PosY: ${height}`);

window.connect("delete-event", () => Gtk.main_quit());
window.show_all();
Gtk.main();